src/app/core/models/templates/sections/test-response-section.model.ts

```diff
@@ -15,12 +23,57 @@ export class TesterResponseSection extends Section {
       while (matches = regex.exec(this.content)) {
         if (matches) {
           const [regexString, title, description, disagreeCheckbox, reasonForDisagreement] = matches;
+
+          if (this.isSeverityDispute(title)) {
+            this.teamChosenSeverity = this.parseTeamChosenSeverity(description);
+          } else if (this.isTypeDispute(title)) {
+            this.teamChosenType = this.parseTeamChosenType(description);
+          }
+
           this.testerResponses.push(new TesterResponse(title, description, disagreeCheckbox, reasonForDisagreement.trim()));
         }
       }
     }
   }
 
+  isSeverityDispute(title: string): boolean {
+    return title.trim() === this.ISSUE_SEVERITY_DISPUTE;
+  }
+
+  isTypeDispute(title: string): boolean {
+    return title.trim() === this.ISSUE_TYPE_DISPUTE;
+  }
+
+  getTeamChosenType(): string {
+    return this.teamChosenType;
+  }
+
+  getTeamChosenSeverity(): string {
+    return this.teamChosenSeverity;
+  }
+
+  parseTeamChosenValue(description: string, prefix: string): string {
```

Comment:
This function can be considered a helper function that grabs a substring that occurs between two specified strings.
Hence, it is not very strongly related to parsing the tester response, and we can move it into a helper module.

The signature can be `extractStringBetween(text: string, prefix: string, suffix: string)`, and we can then just use this function in `parseTeamChosenSeverity` and `parseTeamChosenType`

Reply:
Sure, I chose this method signature as I could not think of a suitable name for it. So we can go ahead with you suggestion to make it more general and reusable 👍